jumpscale/packages/vdc_dashboard/package.toml

```diff
@@ -37,15 +36,12 @@ port = 5591
 
 [[servers]]
 name = "vdc_dashboard_root_proxy"
-domain = "vdc-dashboard.threefold.me"
+domain = "mazraa-dgbtests.vdctest.grid.tf"
```

Comment:
please reset this file to how it was

Reply:
Oh, wasn't modified by me, will see

Reply:
Restored the original version and it is excluded now